### ISSUE TITLE
Create ComplexDialogBehavior that does not use aria-describedby to point to the whole content

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluent-ui-react#1367]
### Problem description
DialogBehavior currently adds `aria-describedby` attribute and makes a connection to the `content` slot. This is useful for confirmation dialogs. But for dialogs that have focusable elements within the content, it might be overwhelming for the user to get the whole content read out.

### Proposed solution
Create `ComplexDialogBehavior` that adds the same attributes, but omits `aria-describedby`. Explain in the docsite which should be used when.
 
### MVP
[Fork, update, and replace this pen if you can show the proposed solution]:
https://codesandbox.io/s/2l3n74j9y

**Comment by jurokapsiar** [comment]
User is able to override aria-describedby prop istead.
